rclcpp_components/CMakeLists.txt

```diff
@@ -41,6 +41,15 @@ ament_target_dependencies(component_container
   "rclcpp"
 )
 
+set(install_dir ${PROJECT_BINARY_DIR})
+configure_file(rclcpp_components_template_node.cmake.in
+  ${CMAKE_CURRENT_SOURCE_DIR}/cmake/rclcpp_components_register_node.cmake @ONLY)
+
+install(FILES 
+  src/node_main.cpp.in 
+  DESTINATION ${PROJECT_BINARY_DIR}
```

Comment:
This should be replaced with the value set previously, e.g. `${node_main_template_install_dir}`.